azure-pipelines.release.yml

```diff
@@ -0,0 +1,64 @@
+trigger:
+  branches:
+    include:
+      - release-*
+
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: 1ESPT-Ubuntu22.04
+      os: linux
+
+    stages:
+      - stage: buildStage
+        displayName: Build Stage
+        jobs:
+          - job: build
+            displayName: Build
+            steps:
+              - checkout: self
+                clean: true
+
+              - task: NodeTool@0
+                inputs:
+                  versionSpec: 20.x
+                displayName: 'Install Node'
+
+              - script: |
+                  which jq && npm install -g `jq -r '.packageManager' < package.json`
```

Comment:
Not super clear what software packages are available on the machines. `jq` may not be available.